index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Microverse</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <script
+      src="https://kit.fontawesome.com/e520ead38a.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="top">
+        <i class="fa-brands fa-facebook-f social"></i>
+        <i class="fa-brands fa-twitter social"></i>
+        <ul class="top-list">
+          <li class="top-items">English</li>
+          <li class="top-items">MyPage</li>
+          <li class="top-items">Logout</li>
+        </ul>
+      </div>
+      <div class="main-nav">
+        <img src="./images/microverseLogo.png" class="logo" alt="Microverse Logo">
+        <ul class="menu-mobile" id="menu">
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="./about.html">About</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Program</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Join</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">Sponsor</a></li>
+          <li class="mobile-menu-list" onclick="closeMenu()"><a href="#">News</a></li>
+          <i class="fa-solid fa-xmark close" onclick="closeMenu()"></i>
```

Comment:
- [ ] Please wrap this `<i>` tag in a `<li>` tag as it will make it semantically more correct. Please refer to this [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) to get more information on which elements are allowed inside a `<ul>` tag. ✍️

```
<ul>
  ...
  <li><i></i></li>
</ul>
```

Reply:
Fixed.